examples/user_guide/Network_Graphs.ipynb

```diff
@@ -0,0 +1,335 @@
+    "#### A simple Graph\n",
+    "\n",
+    "Let's start by declaring a very simple graph connecting one node to all others. If we simply supply the abstract connectivity of the ``Graph``, it will automatically compute a layout for the nodes using the ``layout_nodes`` operation, which defaults to a circular layout:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Declare abstract edges\n",
+    "N = 8\n",
+    "node_indices = np.arange(N)\n",
+    "source = np.zeros(N)\n",
+    "target = node_indices\n",
+    "\n",
+    "padding = dict(x=(-1.2, 1.2), y=(-1.2, 1.2))\n",
+    "\n",
+    "simple_graph = hv.Graph(((source, target),)).redim.range(**padding)\n",
+    "simple_graph"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Accessing the nodes and edges\n",
+    "\n",
+    "We can easily access the ``Nodes`` and ``EdgePaths`` on the ``Graph`` element using the corresponding properties:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "simple_graph.nodes + simple_graph.edgepaths"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Supplying explicit paths\n",
+    "\n",
+    "Next we will extend this example by supplying explicit edges:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def bezier(start, end, control, steps=np.linspace(0, 1, 100)):\n",
+    "    return (1-steps)**2*start + 2*(1-steps)*steps*control+steps**2*end\n",
+    "\n",
+    "x, y = simple_graph.nodes.array([0, 1]).T\n",
+    "\n",
+    "paths = []\n",
+    "for node_index in node_indices:\n",
+    "    ex, ey = x[node_index], y[node_index]\n",
+    "    paths.append(np.column_stack([bezier(x[0], ex, 0), bezier(y[0], ey, 0)]))\n",
+    "    \n",
+    "bezier_graph = hv.Graph(((source, target), (x, y, node_indices), paths)).redim.range(**padding)\n",
+    "bezier_graph"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Interactive features"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Hover and selection policies\n",
+    "\n",
+    "Thanks to Bokeh we can reveal more about the graph by hovering over the nodes and edges. The ``Graph`` element provides an ``inspection_policy`` and a ``selection_policy``, which define whether hovering and selection highlight edges associated with the selected node or nodes associated with the selected edge, these policies can be toggled by setting the policy to ``'nodes'`` and ``'edges'``."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%opts Graph [tools=['hover']]\n",
+    "bezier_graph.opts(plot=dict(inspection_policy='edges'))"
+   ]
```

Comment:
Can we have hover info about the edge? I don't see an example of that.

Reply:
Not currently, another issue to raise with bokeh.

Reply:
And these are issues you haven't already raised?

Reply:
No, we still don't have a way of efficiently storing edge information in HoloViews (although that is work I'll be doing for 1.9), so I haven't gotten around to this yet and won't in this PR.

Reply:
Ok. Please raise a separate issue for that and reference it in this PR, but then leave it open.

Reply:
Nevermind, the ``inspection_policy`` determines whether you can inspect nodes or edges and you can just add the edge information to the abstract graph edges, so this works now.